src/sentry/api/bases/organization.py

```diff
@@ -148,7 +149,7 @@ def get_project_ids(
                 pass
 
         if (
-            request.user.is_superuser
+            user and user.is_superuser
             or (om_role and roles.get(om_role).is_global)
```

Comment:
hmm i'm wondering if we need to have a check for something like `has_valid_api_key` (from below) here otherwise, will this just wind up returning no projects for requests sent via the client?

Reply:
This should be handled by `force_global_perms=True` for the release endpoints. I need to investigate further to understand the cases where user is None